### PR TITLE
WRO-7542: Removed `web-animations-js` dependency + updated enact dependencies

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Removed `web-animations-js` dependency - it was causing an error on isomophic build and is redundant since 2020 on IOS
Updated enact dependencies

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-7542

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: tutorial-typescript/src/components/Counter/Counter.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {adaptEvent, forward, handle} from '@enact/core/handle';
 import kind from '@enact/core/kind';
 import Button from '@enact/sandstone/Button';
 import Changeable from '@enact/ui/Changeable';
+import PropTypes from 'prop-types';
```
Comment (0x64) [inline tutorial-typescript/src/components/Counter/Counter.tsx:5]: This file is updated in #176, so please sync with `develop` branch.

File: tutorial-typescript/types/react-app-env.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-/// <reference types="node" />
-/// <reference types="react" />
-/// <reference types="react-dom" />
+// / <reference types="node" />
```
Comment (0x64) [inline tutorial-typescript/types/react-app-env.d.ts:1]: These are [Triple-Slash Directives](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html), so there should be no space in triple-slash.

**Review by 0x64 — COMMENTED** [review]
@daniel-stoian-lgp Could you check my comments? Thank you.

File: tutorial-typescript/types/react-app-env.d.ts (hunk 2 above)
Comment (daniel-stoian-lgp) [reply tutorial-typescript/types/react-app-env.d.ts]: fixed

**Review by 0x64 — APPROVED** [review]

**Pull request merged**